.travis.yml

```diff
@@ -46,14 +46,21 @@ matrix:
     fast_finish: true
 
     include:
+        # Testing remote-data when running builds on master or with
+        # cron. Starting these early as they take much longer to run.
+        - os: linux
+          env: EVENT_TYPE='push cron' DEBUG=True
```

Comment:
I've missed to add the remotes here, but have it in locally and this will anyway need a rebase to remove the temporary commit.